### DISCUSSION TITLE
[Windows][NET6] Doesn't compile when using DropDownPreferredPlacement for mobile

### DISCUSSION THREAD

**Discussion by Soap-141** [unoplatform/uno#11797] [Q&A] [answered]
Using Uno documentation [here](https://platform.uno/docs/articles/controls/ComboBox.html?q=DropDownPreferredPlacement) I managed to create a `ComboBox` style using `DropDownPreferredPlacement` and it compile on mobile but not on Windows.

This is what I'm using.

`xmlns:uno_controls="using:Uno.UI.Xaml.Controls"`
`<uno_controls:Setter Property="ComboBox.DropDownPreferredPlacement" Value="Below" />`

This is the error I get on Windows.
`1>Styles\MergedResourceDictionaries.xaml(1442,73): XamlCompiler error WMC0610: The XAML Binary Format (XBF) generator reported syntax error '0x09c6'`

Is there a way to make it compile? Am I missing something here?

**Comment by jeromelaban** [comment]
You'll need to place `uno_controls` in the ignorable list, otherwise WinAppSDK will try to read it.

**Reply by Soap-141** [reply]
Sure it was in ignore, I'll double check soon and give you an update.

**Reply by Soap-141** [reply]
It's already in `mc:Ignorable="ios xamarin uno_controls"`.

**Reply by Soap-141** [reply]
It's already in `mc:Ignorable="ios xamarin uno_controls"`.

**Reply by jeromelaban** [reply]
If it's already there, it means that Windows is still trying to read your style, somehow and it's not something Uno can adjust. You may want to duplicate the style and set `uno_controls` on the style, and not setters.

**Comment by Soap-141** [comment, marked as answer]
Like @jeromelaban said, I had to create a style for Windows and a style for mobile. I created a base style to not have duplicated code, created the two styles that are based on the base style and added the specific line for mobile only.
